examples/serialrx/Kconfig

```diff
@@ -39,22 +39,22 @@ config EXAMPLES_SERIALRX_DEVPATH
 
 choice
 	prompt "Output method"
-	default EXAMPLES_SERIALRX_PRINTHYPHEN
+	default EXAMPLES_SERIALRX_PRINTSTR
```

Comment:
why change the default setting

Reply:
When I opened this application for the first time, it was very confusing for me to not have read the source code of the program and not have any display after using another serial port to send data, after all, there is no explanation in the HELP. So I would like to change the default value to EXAMPLES_SERIALRX_PRINTSTR so that later learners can be more intuitively clear that the previous operation is correct.